<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>signUp</class>
 <widget class="QMainWindow" name="signUp">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>980</width>
    <height>709</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:#d3ffc5;   /* Mint green */</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>270</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{font-size:18px;}</string>
    </property>
    <property name="text">
     <string>Confirm password:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_5">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>260</y>
      <width>331</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
    font-size: 10px;                   /* Default font size */
    color: #000000;                    /* Black text color */
    padding: 8px;                      /* Inner padding */
    background-color: #f9f9f9;        /* Light gray background */
    border: 2px solid #cccccc;         /* Light gray border */
    border-radius: 5px;                /* Rounded corners */
}

QLineEdit:focus {
    border: 2px solid #4CAF50;         /* Green border when focused */
    background-color: #ffffff;         /* White background when focused */
}

QLineEdit::placeholder {
    color: #aaaaaa;                    /* Light gray for placeholder text */
    font-style: italic;                /* Italic style for a subtle hint */
    font-size: 16px;                   /* Placeholder font size */
}
</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>330</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{font-size:18px;}</string>
    </property>
    <property name="text">
     <string>Email:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>210</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{font-size:18px;}</string>
    </property>
    <property name="text">
     <string>Enter Password:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_3">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>390</y>
      <width>331</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
    font-size: 10px;                   /* Default font size */
    color: #000000;                    /* Black text color */
    padding: 8px;                      /* Inner padding */
    background-color: #f9f9f9;        /* Light gray background */
    border: 2px solid #cccccc;         /* Light gray border */
    border-radius: 5px;                /* Rounded corners */
}

QLineEdit:focus {
    border: 2px solid #4CAF50;         /* Green border when focused */
    background-color: #ffffff;         /* White background when focused */
}

QLineEdit::placeholder {
    color: #aaaaaa;                    /* Light gray for placeholder text */
    font-style: italic;                /* Italic style for a subtle hint */
    font-size: 16px;                   /* Placeholder font size */
}
</string>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::EchoMode::Normal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>150</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{font-size:18px;}</string>
    </property>
    <property name="text">
     <string>Full name:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>70</y>
      <width>321</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
    font-size: 28px;                 /* Larger font size for the header */
    font-weight: bold;               /* Make the text bold */
    color: #1a73e8;                  /* Blue color for a modern look */
    padding: 12px;                   /* Padding around the text */
    background-color: #e6f2ff;      /* Light blue background for contrast */
    border-radius: 10px;             /* Rounded corners */
    text-align: center;              /* Center the text */
    letter-spacing: 1px;             /* Slightly increase letter spacing */
}
</string>
    </property>
    <property name="text">
     <string>Create your account</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Login">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>480</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    font-size: 18px;
    padding: 10px 20px;                /* Vertical and horizontal padding */
    color: #ffffff;                     /* White text color */
    background-color: #4CAF50;         /* Green background for login */
    border: none;                       /* No border */
    border-radius: 5px;                /* Rounded corners */
    transition: background-color 0.3s;  /* Smooth transition effect */
}</string>
    </property>
    <property name="text">
     <string>login</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>10</y>
      <width>181</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
    font-size: 32px;            /* Larger font size for emphasis */
    color: #4CAF50;             /* Green color for a fresh look */
    font-weight: bold;          /* Make the text bold */
    padding: 10px 20px;         /* Add padding around the text */
    background-color: #e8f5e9;  /* Light green background to match the color scheme */
    border-radius: 8px;         /* Rounded corners */
    text-align: center;         /* Center-align the text */
}
</string>
    </property>
    <property name="text">
     <string>Sign up</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>140</y>
      <width>331</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
    font-size: 10px;                   /* Default font size */
    color: #000000;                    /* Black text color */
    padding: 8px;                      /* Inner padding */
    background-color: #f9f9f9;        /* Light gray background */
    border: 2px solid #cccccc;         /* Light gray border */
    border-radius: 5px;                /* Rounded corners */
}

QLineEdit:focus {
    border: 2px solid #4CAF50;         /* Green border when focused */
    background-color: #ffffff;         /* White background when focused */
}

QLineEdit::placeholder {
    color: #aaaaaa;                    /* Light gray for placeholder text */
    font-style: italic;                /* Italic style for a subtle hint */
    font-size: 16px;                   /* Placeholder font size */
}
</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>470</y>
      <width>281</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
    font-size: 22px;              /* Increase the font size */
    color: #1a73e8;               /* Set the text color to blue */
    font-weight: bold;            /* Make the text bold */
    padding: 5px;                 /* Add padding around the text */
}
</string>
    </property>
    <property name="text">
     <string>Already have an account?</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>400</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{font-size:18px;}</string>
    </property>
    <property name="text">
     <string>Phone number:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_2">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>320</y>
      <width>331</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
    font-size: 10px;                   /* Default font size */
    color: #000000;                    /* Black text color */
    padding: 8px;                      /* Inner padding */
    background-color: #f9f9f9;        /* Light gray background */
    border: 2px solid #cccccc;         /* Light gray border */
    border-radius: 5px;                /* Rounded corners */
}

QLineEdit:focus {
    border: 2px solid #4CAF50;         /* Green border when focused */
    background-color: #ffffff;         /* White background when focused */
}

QLineEdit::placeholder {
    color: #aaaaaa;                    /* Light gray for placeholder text */
    font-style: italic;                /* Italic style for a subtle hint */
    font-size: 16px;                   /* Placeholder font size */
}
</string>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::EchoMode::Normal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="clear">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>540</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    font-size: 18px;
    padding: 10px 20px;                /* Vertical and horizontal padding */
    color: #ffffff;                     /* White text color */
    background-color: #4CAF50;         /* Green background for login */
    border: none;                       /* No border */
    border-radius: 5px;                /* Rounded corners */
    transition: background-color 0.3s;  /* Smooth transition effect */
}</string>
    </property>
    <property name="text">
     <string>clear</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_4">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>200</y>
      <width>331</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
    font-size: 10px;                   /* Default font size */
    color: #000000;                    /* Black text color */
    padding: 8px;                      /* Inner padding */
    background-color: #f9f9f9;        /* Light gray background */
    border: 2px solid #cccccc;         /* Light gray border */
    border-radius: 5px;                /* Rounded corners */
}

QLineEdit:focus {
    border: 2px solid #4CAF50;         /* Green border when focused */
    background-color: #ffffff;         /* White background when focused */
}

QLineEdit::placeholder {
    color: #aaaaaa;                    /* Light gray for placeholder text */
    font-style: italic;                /* Italic style for a subtle hint */
    font-size: 16px;                   /* Placeholder font size */
}
</string>
    </property>
   </widget>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>10</y>
      <width>231</width>
      <height>141</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget{
border-image:url(:/piyu/4.jpg);
}
background-color:#d3ffc5;   /* Mint green */</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Sign_Up">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>540</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    font-size: 18px;
    padding: 10px 20px;                /* Vertical and horizontal padding */
    color: #ffffff;                     /* White text color */
    background-color: #4CAF50;         /* Green background for login */
    border: none;                       /* No border */
    border-radius: 5px;                /* Rounded corners */
    transition: background-color 0.3s;  /* Smooth transition effect */
}</string>
    </property>
    <property name="text">
     <string>sign up</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>980</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
